lib/web_ui/lib/util.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Can we add this code to `initialization.dart`? Seems like all these are related to initializing apps.

Reply:
Done.